.github/workflows/build_master.yml

```diff
@@ -0,0 +1,56 @@
+name: build_master
+on:
+  push:
+    branches:
+      - master
+jobs:
+  after_master_commit:
+    name: Build after master commit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+      - uses: subosito/flutter-action@v1
+        with:
+          flutter-version: '3.0.5'
+          channel: 'stable'
+      - name: Create api key for android
+        run: |
+          cd android;
+          touch "local.properties";
+          echo "${{secrets.FACEBOOK_APP_ID}}" >> local.properties;
+          echo "${{secrets.FB_LOGIN_PROTOCOL_SCHEME}}" >> local.properties;
+          echo "${{secrets.FACEBOOK_CLIENT_TOKEN}}" >> local.properties;
+      - name: Create api key
+        run: |
+          touch "prod.json";
+          touch "sandbox.json";
```

Comment:
why do u create sandbox.json? seems that we don't need it for this build

Reply:
its use is written in the assets, without this option - an error